content/news/posts/2025/breaking-change-guzzle-removal-v9.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title:  "Breaking Change in V9 Guzzle removal, what are the alternatives?"
```

Comment:
```suggestion
title:  "Migrating to PrestaShop 9: Guzzle is gone, what's next?"
```

Reply:
I'm also thinking about changing it to "Adapting to PrestaShop 9" 🤔

Reply:
I'll replace MIgrating with Adapting